readinglist/backend/__init__.py

```diff
@@ -36,6 +36,16 @@ def get_all(self, resource, user_id, filters=None, sorting=None,
                 pagination_rules=None, limit=None):
         raise NotImplementedError
 
+    def strip_deleted_record(self, resource, user_id, record):
+        """Strip the record to its deleted version (in place)."""
```

Comment:
Okay, I think I got it: so when we delete something, we add a deletion marker on it and remove all the information associated with the record in the database.